frontend/website/pages/docs/developers/sandbox-node.mdx

```diff
@@ -39,6 +39,14 @@ And stop coda by running.
 docker stop coda
 ```
 
+You can use the Coda CLI to interact with the sandbox node. The following command opens a shell inside the docker container from where you can issue any of the available [coda commands](/docs/cli-reference).
+
+```
+docker exec -it coda bash
+```
+
+Now you can execute coda commands like normal.
```

Comment:
this is already present next to `coda client status` in the docs, but I think it's good to call it out here too

Reply:
good catch